internal/config/rules.go

```diff
@@ -0,0 +1,44 @@
+package config
```

Comment:
Let's put this in `internal/mediators/config`, the idea being that `mediators` is what will eventually be the `internal` directory for all our commands (whereas currently it is obfuscated with a lot of `cmd/state` specific stuff.